catalogue/admin.py

```diff
@@ -419,6 +420,46 @@ def component_link(self, obj):
         )
 
 
+class ComponentLogInline(admin.TabularInline):
+    model = ComponentLog
+    ordering = ("-created",)
+    can_delete = False
+    fields = (
+        "created",
+        "component",
+    )
+    readonly_fields = ("component", "created")
+    extra = 0
+
+    def has_add_permission(self, request, obj=None):
+        return False
+
+
+@admin.register(SearchLog)
+class SearchLogAdmin(admin.ModelAdmin):
+    list_display = (
+        "identifier",
+        "created",
+        "get_query",
+        "get_comp_count",
+    )
+    search_fields = (
+        "query",
+        "created",
+        "identifier",
+    )
+    readonly_fields = ("query", "created", "identifier")
+    inlines = [ComponentLogInline]
+
+    @admin.display(description=SearchLog._meta.get_field("query").verbose_name)
+    def get_query(self, obj):
+        return mark_safe(f"<a href='{obj.query}'>{obj.query}</a>")
+
+    @admin.display(description="Anzahl angeklickter Lösungen")
+    def get_comp_count(self, obj):
+        return ComponentLog.objects.filter(query=obj.id).count()
```

Comment:
use `annotate()` to be able to specify `admin_order_field` here. https://stackoverflow.com/a/4547426